app/views/devise/passwords/new.html.erb

```diff
@@ -1,3 +1,5 @@
+<% content_for :title, "Forgot password" -%>
```

Comment:
I thought about that but I think it's too long. Remember that ` - Huginn` is appended to each title, so we should try to keep them as short as possible.